lib/config/config-file.js

```diff
@@ -176,6 +176,36 @@ function loadPackage(filePath) {
 }
 
 /**
+ * Loads a JavaScript configuration from a plugin.
+ * @param {string} filePath The plugin name to load.
+ * @returns {Object} The configration object from the plugin.
+ * @throws {Error} If the package cannot be read.
+ * @private
+ */
+function loadPlugin(filePath) {
+    var parts = filePath.split("/"),
+        pluginName = parts[0],
+        configName = parts[1];
+    debug("Loading config plugin: " + filePath);
+    try {
+        var plugin = require(pluginName);
+        if (!plugin.configs) {
+            debug("Error reading package: " + filePath);
+            throw new Error("Plugin " + pluginName + " does not expose any configs");
+        }
+        if (configName && !plugin.configs[configName]) {
+            debug("Error reading package: " + filePath);
+            throw new Error("Plugin " + pluginName + " is missing specified configuration");
```

Comment:
Including `configName` in the error message might help users track down configuration issues.